website/content/docs/bloop-basics.md

```diff
@@ -0,0 +1,127 @@
++++
+toc = true
+weight = 2
+draft = false
+title = "The basics of bloop"
+date = "2018-02-09T10:15:00+01:00"
+description = "What is bloop and which problems does it solve?"
+bref = "Learn the ideas behind bloop, which problems solve and how it's designed."
```

Comment:
which problems it solves? are solved?

Reply:
Hot compilers, et al. I think the document does address that.